README.md

```diff
@@ -12,12 +12,53 @@ information about installing the `polkadot` binary and developing on the codebas
 specific guides, like how to be a validator, see the
 [Polkadot Wiki](https://wiki.polkadot.network/docs/en/).
 
-## Building
+## Installation
+
+If you just wish to run a Polkadot node without compiling it yourself, you may
+either run the latest binary from our
+[releases](https://github.com/paritytech/polkadot/releases) page, or install
+Polkadot from one of our package repositories.
+
+Installation from the debian or rpm repositories will create a `systemd`
+service that can be used to run a Polkadot node. This is disabled by default,
+and can be started by running `systemctl start polkadot`. By default, it will
+run as the `polkadot` user.  Command-line flags passed to the binary can be
+customised by editing `/etc/default/polkadot`. This file will not be
+overwritten on updating polkadot. You may also just run the node directly from
+the command-line.
+
+### Debian-based (Debian, Ubuntu)
+
+Currently supports Debian 10 (Buster) and Ubuntu 20.04 (Focal), and
+derivatives. Run the following commands as the `root` user.
+
+```
+# Import the security@parity.io GPG key
+gpg --recv-keys --keyserver hkps://keys.mailvelope.com FF0812D491B96798
```

Comment:
This needs to be a full (40 hex character) fingerprint, to prevent preimage attacks.  If `keys.mailvelope.com` is a pool, we should also point to an individual server, as GPG can’t receive keys from a pool if it cannot resolve DNS.

Reply:
40-hex-character fingerprint = :+1: 

Could you clarify on your second point? Do you mean specifying the keyserver with a single IP address in case of DNS failure? If so, is that a better solution than not being able to receive keys in the event of DNS issues?

Reply:
GPG does support using an HTTP proxy, but only for standalone keyservers.  It cannot receive keys from a pool without DNS.  This has caused problems for me in the past when installing software in QubesOS, since QubesOS TemplateVMs can only access the Internet via a proxy and have no ability to resolve DNS.

Reply:
GPG does support using an HTTP proxy, but only for standalone keyservers.  It cannot receive keys from a pool without DNS.  This has caused problems for me in the past when installing software in QubesOS, since QubesOS TemplateVMs can only access the Internet via a proxy and have no ability to resolve DNS.